rdkit/RDConfig.py

```diff
@@ -29,10 +40,13 @@
   if condaDir[0]=='':
     condaDir[0] = os.path.sep
   condaDir += ['share','RDKit']
-  _share = os.path.join(*condaDir)
-  RDDataDir=os.path.join(_share,'Data')
-  RDDocsDir=os.path.join(_share,'Docs')
-  RDProjDir=os.path.join(_share,'Projects')
+  RDBaseDir = os.path.join(*condaDir)
+  RDDataDir=os.path.join(RDBaseDir,'Data')
+  RDDocsDir=os.path.join(RDBaseDir,'Docs')
+  RDProjDir=os.path.join(RDBaseDir,'Projects')
+  RDBinDir=MissingCondaDir('bin')
+  RDDemoDir=MissingCondaDir('Demo')
+  RDCodeDir=MissingCondaDir('rdkit')
```

Comment:
It seems like in a conda environment (or when RDBaseDir == /usr/local/share after a brew install for instance) $RDBASE/rdkit doesn't exist.  This is only true in the in-tree build unless I'm mistaken.  If we want access to the source tree, the following should always be correct:

RDConfig.py ...
  RDCodeDir=os.path.dirname(__file__)